src/ui/popup/settings-page/devtools.tsx

```diff
@@ -7,14 +7,27 @@ import {ViewProps} from '../types';
 import {isFirefox} from '../../../utils/platform';
 
 function getExistingDevToolsWindow() {
-    return new Promise<chrome.windows.Window>((resolve) => {
-        chrome.windows.getAll({
-            populate: true,
-            windowTypes: ['popup']
-        }, (w) => {
-            for (const window of w) {
-                if (window.tabs[0].url.endsWith('ui/devtools/index.html')) {
-                    resolve(window);
+    if (!isMobile()) {
```

Comment:
Make this line `if (isMobile()) {`, so that previously working code is left unchanged.